Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -213,30 +214,46 @@ public ActionResult<IEnumerable<TeamExtendedViewModel>> GetTeamInvites()
         /// Accept one specified team invite and true delete others from the same activity if there's any
         /// </summary>
         /// <param name="teamID"></param>
-        /// <param name="acceptedInvite"></param>
+        /// <param name="response"></param>
         /// <returns>The accepted TeamInviteViewModel</returns>
         [HttpPatch]
-        [Route("{teamID}/invite")]
-        public async Task<ActionResult<ParticipantTeamViewModel>> AcceptTeamInvite(int teamID, [FromBody] ParticipantTeamUploadViewModel acceptedInvite)
+        [Route("{teamID}/invite/{response}")]
+        public async Task<ActionResult<TeamInviteViewModel>> AcceptTeamInvite(int teamID, string response)
         {
             var username = AuthUtils.GetUsername(User);
             try
             {
                 var invite = _teamService.GetTeamInvite(teamID, username);
                 if (invite is null)
                     return NotFound("You were not invited by this team.");
-                if (username != invite.ParticipantUsername)
-                    return Forbid($"You are not permitted to accept invitations for another participant.");
```

Comment:
Why did you remove this. Please see my comments in #809 about the security vulnerability.

Reply:
I added this back in, I was not sure how AuthUtils could get the wrong username for the one who was invited, but it is a good check to make sure.

Reply:
Wait I'm now confused. How would you payload someone else's username when it's not being passed anywhere? I can't even swagger it without manually adjusting the route.

Reply:
No you're right, I think I'm confusing myself at this point.

Reply:
hahaha all good :)

Reply:
I think @bennettforkner got confused about where the `invite` is coming from.

In this case, the invite is being retrieved from the database using the authenticated user's username, so the vulnerability he described does not apply.

He was thinking that `invite` is being passed into the route as a Request body, which would open the vulnerability he described.